content/en/apps/features/supervision/supervision.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Supervision
+weight: 7
+description: >
+  Supervision and workforce management to strengthen health systems
+keywords: supervisor
+relatedContent: >
+  apps/features/targets
+  design/best-practices/#targets
+  apps/features/integrations/dhis2
+---
+
+Supervision and workforce management are important aspects to building and maintaining high performing community health systems. Supervisors help Community Health Workers to deliver quality healthcare services to their patients through: 1) building CHW care delivery knowledge and skills; 2) fostering a supportive work environment; and 3) supporting continuity of care between home-based care and community health centers or facilities. 
+
+The WHO’s [guidelines](https://www.who.int/hrh/community/guideline-health-support-optimize-hw-programmes/en/) on health policy and system support to optimize community-based health worker programmes emphasizes the importance of regular and systematic CHW supervision. Following these guidelines, the CHT is designed to enable Supervisors to provide personalized performance feedback during CHW supervision as well as track aggregate statistics. Data collected from completed [Care Guides]({{< relref "apps/concepts/care-guides" >}}) produce granular information that can be used to provide coaching to promote compliance with health program standards of practice and monitor outcomes.
```

Comment:
```suggestion
The WHO’s [guidelines](https://www.who.int/hrh/community/guideline-health-support-optimize-hw-programmes/en/) on health policy and system support formulated to optimize community-based health worker programmes emphasizes the importance of regular and systematic CHW supervision. Following these guidelines, the CHT is designed to enable Supervisors to provide personalized performance feedback during CHW supervision as well as track aggregate statistics. Data collected from completed [Care Guides]({{< relref "apps/concepts/care-guides" >}}) produce granular information that can be used to provide coaching that promotes compliance with health program standards of practice and closer monitoring of outcomes.
```

As a layperson, I had trouble with the phrase "health policy and system support to optimize community-based". I verified the intended meaning by looking at the linked article, but thought adding "formulated" helps a bit with the sentence flow (so "system support" appears more clearly as a separate noun)

The other changes may also be due to me being a layperson ;)